cmd/juju/charmhub/find.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package charmhub
+
+import (
+	"fmt"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/charmhub"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+const (
+	findSummary = "Queries the charmhub store for available charms or bundles."
+	findDoc     = `
+The find command queries the charmhub store for available charms of bundles.
+
+Examples:
+    juju find wordpress
+
+See also:
+    info
+`
+)
+
+// NewFindCommand wraps findCommand with sane model settings.
+func NewFindCommand() cmd.Command {
+	return modelcmd.Wrap(&findCommand{})
+}
+
+// findCommand supplies the "find" CLI command used to display find information.
+type findCommand struct {
+	modelcmd.ModelCommandBase
+
+	api FindCommandAPI
+
+	verbose bool
```

Comment:
Where is this var used?  I believe we get --verbose for free from ModelCommandBase.